x-pack/plugins/maps/public/components/metrics_editor/metric_editor.tsx

```diff
@@ -85,6 +99,16 @@ export function MetricEditor({
       field: fieldName,
     });
   };
+
+  const onPercentileChange = (e: ChangeEvent<HTMLInputElement> | MouseEvent<HTMLButtonElement>) => {
```

Comment:
Would it be possible to store percentile in local state and have a debounced call to `onChange `? The slider UI component has a laggy feel since the updates must do a round trip to redux.

Reply:
Maybe adding the number input is good enough so that its easy to set exact percentiles.